pkg/k8shandler/util.go

```diff
@@ -262,7 +263,7 @@ func (er *ElasticsearchRequest) isValidConf() error {
 		if err := updateInvalidUUIDChangeCondition(dpl, v1.ConditionFalse, "", client); err != nil {
 			return err
 		}
-	}
+	}*/
```

Comment:
Can you just delete this?

Reply:
nack, i need to uncomment this before its merged in. i needed to comment this out to be able to recreate a condition from 4.5